docs/setup/pi4.md

```diff
@@ -0,0 +1,47 @@
+# Connect Create® 3 to Raspberry Pi® 4 and set up ROS 2 Galactic
```

Comment:
should we link to the other page where you show how to connect RPI4 to the create?

Reply:
Added link in 76164cbd58a389797cf0dc1f3fe42fe21a7d178c